zebra/zebra_mpls_vty.c

```diff
@@ -247,7 +247,7 @@ DEFUN (mpls_label_bind,
 
 DEFUN (no_mpls_label_bind,
        no_mpls_label_bind_cmd,
-       "no mpls label bind <A.B.C.D/M|X:X::X:X/M> [<(16-1048575)|implicit-null>]",
+       "no mpls label bind <A.B.C.D/M|X:X::X:X/M> [<(16-1048575)|implicit-null|explicit-null>]",
```

Comment:
docstring must be added also for explicit-null.

Reply:
Done, thanks!